README.md

```diff
@@ -66,8 +65,20 @@ Commands:
   tx2g  This command loads DiVA database transactions into a graph
 ```
 
-## Links
-Link to complete [Documentation](https://konveyor.github.io/tackle-data-gravity-insights/) of the project
+## Demo
+
```

Comment:
Please add back fully qualified URLs so that this readme continues to work as the home page on PyPi.